Library/UIAlertController.swift

```diff
@@ -356,4 +356,37 @@ public extension UIAlertController {
 
     return alertController
   }
+
+  static func blockUserActionSheet(
+    viewProfileHandler: ((UIAlertAction) -> Void)? = nil,
+    blockUserHandler: ((UIAlertAction) -> Void)? = nil,
+    sourceView: UIView? = nil
+  ) -> UIAlertController {
+    let alertController = UIAlertController(title: nil, message: nil, preferredStyle: .actionSheet)
+
+    if let profileHandler = viewProfileHandler {
+      // Scott TODO: Use localized strings once translations can be done [mbl-1037](https://kickstarter.atlassian.net/browse/MBL-1037)
+      alertController
+        .addAction(UIAlertAction(title: "View Profile", style: .default, handler: profileHandler))
+    }
+
+    // Scott TODO: Use localized strings once translations can be done [mbl-1037](https://kickstarter.atlassian.net/browse/MBL-1037)
+    alertController
+      .addAction(UIAlertAction(title: "Block this user", style: .destructive, handler: blockUserHandler))
+
+    alertController
+      .addAction(UIAlertAction(title: Strings.Cancel(), style: .cancel, handler: nil))
+
+    if AppEnvironment.current.device.userInterfaceIdiom == .pad {
+      alertController.modalPresentationStyle = .popover
+      alertController.popoverPresentationController?.sourceView = sourceView
+    }
+
+    // Accessibility
+    // Scott TODO: Use localized strings once translations can be done [mbl-1037](https://kickstarter.atlassian.net/browse/MBL-1037)
+    alertController.view.accessibilityIdentifier = "block_user_action_sheet"
+    alertController.view.accessibilityValue = "User Menu"
```

Comment:
Soo, I don't think adding extra accessibility stuff here actually helped the way I wanted it to. There's still a sound and an "Alert" announced when the model is presented (and voiceover is active). I didn't see an easy way to turn it off, unfortunately. I was hoping that the "Alert" was only read out as a default title, but it's read out before the title if it's provided. I think we can just get rid of the "User menu" title, unless you think we'll have time to look for a fix here.

Reply:
AlertControllers have voice-over support by default, which is the announced "alert" you're referring to. I can remove it for now since this doesn't seem to be replacing it.